src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./blocks/variables.scss";
+@import "./blocks/header.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/catalog.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+p {
   margin: 0;
 }
```

Comment:
Move to normalize files, in main should placed just import

Reply:
moved